lib/lib-dynamodb/src/commands/BatchExecuteStatementCommand.ts

```diff
@@ -104,3 +96,12 @@ export class BatchExecuteStatementCommand extends DynamoDBDocumentClientCommand<
     return async () => handler(this.clientCommand);
   }
 }
+
+import type {
+  BatchExecuteStatementCommandInput as __BatchExecuteStatementCommandInput,
+  BatchExecuteStatementCommandOutput as __BatchExecuteStatementCommandOutput,
+  BatchStatementError,
+  BatchStatementRequest,
+  BatchStatementResponse,
+} from "@aws-sdk/client-dynamodb";
+import type { NativeAttributeValue } from "@aws-sdk/util-dynamodb";
```

Comment:
Are the type imports towards the bottom normal?

Reply:
I believe eslint plugin `simple-import-sort` is doing it's thing.